doc/nrf/doc_styleguide.rst

```diff
@@ -174,18 +125,36 @@ Pay attention to the following rules when highlighting content:
 
        > set VERBOSE=True && set CMAKE_BUILD_PARALLEL_LEVEL=1 && west build -b *board_name*
 
-* GUI
-
+GUI
   * When writing about a sequence of GUI elements, use a > between them.
     For example: "Select :guilabel:`File` > :guilabel:`Save as...`".
 
-* Filenames and titles
-
+Filenames and titles
   * For readability reasons, do not use any code highlighting for titles of other pages or headings when they are mentioned in the text.
     For example, do not write "The ``bl_crypto`` library".
     Use the complete name of the library: "The bootloader crypto library", and always link to the page you mention.
   * If the page you are linking to is mentioned several times in the text, place the link on the first occurrence and on every occurrence where linking to the page is useful.
 
+PCB names
+=========
+
+When referring to specific Printed Circuit Board (PCB) elements, use capitalization.
+For example, Button 1, Switch 3, LED 1.
+
+If you are referring to a generic PCB element, do *not* use capitalization: button, switch, LEDs.
```

Comment:
I would maybe include this information in line 100. And then remove the rest of the information from this section...